src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    if (inversedRobot.hasOwnProperty(robot[key])) {
```

Comment:
it's better to use Object.prototype.hasOwnProperty.call(inversedRobot, robot[key])
[link](https://eslint.org/docs/latest/rules/no-prototype-builtins)

Reply:
Maybe you're right, but I don't know how to work with prototypes yet